services/oauth/v2/common/constants.go

```diff
@@ -0,0 +1,23 @@
+package common
+
+const (
+	CategoryRefreshToken = "REFRESH_TOKEN"
+	// CategoryAuthStatusInactive Identifier to be sent from destination(during transformation/delivery)
+	CategoryAuthStatusInactive = "AUTH_STATUS_INACTIVE"
+	// RefTokenInvalidGrant Identifier for invalid_grant or access_denied errors(during refreshing the token)
+	RefTokenInvalidGrant = "ref_token_invalid_grant"
```

Comment:
These are not being used? Are we supposed to use these later
```suggestion
```

Reply:
These are being used. Made changes to use these instead of earlier references.